modules/Router.php

```diff
@@ -230,11 +236,28 @@ public static function getPageUrl($page, $content)
     }
 
     /**
+     * Method : getSettings
+     * Description : Returns array of template and data for settings page
```

Comment:
@MehaKaushik settings page template is referred here. It's self-explanatory that template is for the settings page. This comment is according to the other comments approved in the same file.